app/assets/javascripts/admin/addon/components/embeddable-host.js

```diff
@@ -1,11 +1,14 @@
 import I18n from "I18n";
-import discourseComputed from "discourse-common/utils/decorators";
+import {
+  default as discourseComputed,
```

Comment:
Can the default imports be changed to, for example in this case:

```suggestion
import discourseComputed, {
```

The explicit `default as` is a pet peeve of mine 😄